src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -347,24 +409,27 @@ export class HostsComponent extends ListWithDetails implements OnInit {
       'tcmu-runner': 'iscsi'
     };
     this.isLoadingHosts = true;
-    this.hostService.list().subscribe(
-      (resp: any[]) => {
-        resp.map((host) => {
-          host.services.map((service: any) => {
-            service.cdLink = `/perf_counters/${service.type}/${encodeURIComponent(service.id)}`;
-            const permission = this.permissions[typeToPermissionKey[service.type]];
-            service.canRead = permission ? permission.read : false;
-            return service;
+    this.sub = this.timerService
```

Comment:
As pero ur previous conversation you need to refactor this as currently it's creating a new subscription for each call.
Also probably you don't need the timerService.